content/en/monitors/types/ci.md

```diff
@@ -20,22 +20,28 @@ further_reading:
 ---
 
 {{< site-region region="gov" >}}
-<div class="alert alert-warning">CI Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+<div class="alert alert-warning">This feature is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
```

Comment:
```suggestion
{{< site-region region="gov" >}}
<div class="alert alert-warning">The CI/CD & Test Monitor is not supported for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
{{< /site-region >}}
```
Just suggest to add the name of the feature and use the new callout so it links to the Site page